Framework/PythonInterface/test/python/plugins/algorithms/ISISIndirectDiffractionReductionTest.py

```diff
@@ -175,6 +175,25 @@ def test_reduction_with_cal_file_osiris_diffspec(self):
         self.assertEqual(red_ws.getAxis(0).getUnit().unitID(), 'dSpacing')
         self.assertEqual(red_ws.getNumberHistograms(), 1)
 
+
+    def test_reduction_with_vandium_iris(self):
+        """
+        Test to ensure that reduction with normalisation by vanadium works
+        """
+        wks = ISISIndirectDiffractionReduction(InputFiles=['IRS26176.RAW'],
+                                               VanadiumFiles=['IRS26173.RAW'],
+                                               Instrument='IRIS',
+                                               Mode='diffspec',
+                                               SpectraRange=[105, 112])
+
+        self.assertTrue(isinstance(wks, WorkspaceGroup), 'Result workspace should be a workspace group.')
+        self.assertEqual(len(wks), 1)
+        self.assertEqual(wks.getNames()[0], 'iris26176_diffspec_red')
+
+        red_ws = wks[0]
+        self.assertEqual(red_ws.getAxis(0).getUnit().unitID(), 'dSpacing')
+        self.assertEqual(red_ws.getNumberHistograms(), 1)
+
```

Comment:
We should check a few random values in this unit test to ensure that the values produced are correct.